site/about/style-guide.qmd

```diff
@@ -0,0 +1,230 @@
+|   *Uncertainty* can be summed up as the difference between reality and the outputs from the model selected to approximate reality.   |  “Uncertainty” can be summed up as the difference between reality and the outputs from the model selected to approximate reality.   |
+|   “ValidMind is the only platform today that is purpose-built for model risk management professionals in the banking industry,” Jacobi says. | At ValidMind, we value "transparency and accessibility"—we aim to speak simply and effectively. |
+: Emphasis examples {.hover}
+
+### Content types
+
+#### Code
+- Format code in its [own code block](https://quarto.org/docs/authoring/markdown-basics.html#source-code).
+- Code language is required.
+
+::: {.callout-tip title="Within a Jupyter Notebook" appearance="minimal"}
+You can simply use a **code cell** rather than a markdown cell.
+:::
+
+> **Correct**:
+> ```python
+>%pip install -q validmind
+>```
+
+| **Incorrect:** 
+|       Using Python, call `%pip install -q validmind`. 
+
+#### Mathematics
+Mathematical formulas should be rendered using **[LaTeX formatting](https://quarto.org/docs/output-formats/html-basics.html#latex-equations)**. On our Wordpress blog posts, this is taken care of by the [WP Quick LaTeX plugin](https://en-gb.wordpress.org/plugins/wp-quicklatex/). 
+
+| Correct | Incorrect | 
+|------|-----|
+|   $likes \sim Binomial(n_{feedbacks},p_{like})$  |  `$likes \sim Binomial(n_{feedbacks},p_{like})$`  |
+: Mathematical formula examples {.hover}
+
+## Proper nouns
+In the context of model risk management, proper nouns include specific models, laws, or regulations, such as "Basel IV" or "SR 11-7." These refer to specific frameworks or guidelines and you spell them with initial capital letters or exactly as indicated by official sources.
+
+- Terms that are not proper nouns include general concepts such as "model validation," "stress testing," "risk assessment," and "backtesting." 
+- These are common terms in the field and are not capitalized unless starting a sentence.
+
+| Correct | Incorrect | 
+|------|-----|
+|   SS1/23 – Model risk management principles for banks  |  Model Validation  |
+|   validation report  |  basel 4  |
+|   machine learning  |  Financial Services industry  |
+: Proper noun examples {.hover}
+
+### Product names
+Within [our documentation](https://docs.validmind.ai/), you are able to reference the {{<var vm_framework>}} and {{<var vm_platform>}} [via variables](https://quarto.org/docs/authoring/variables.html#var). 
+
+- Use the variables shown on the table below instead of writing out the phrases to enable consistency between guides. 
+- If product names need to be updated, simply amend [the `_variables.yml` file](https://github.com/validmind/documentation/blob/main/site/_variables.yml) to see changes reflected throughout all guides. 
+
+| Product Name | Variable Key |
+|------:|-----|
+| {{<var vm_framework>}} |  `{{{<var vm_framework>}}}` |
```

Comment:
Lines 209 - 212 do not render correctly. It looks like you need to add spaces after `<` and before `>`.

Reply:
Really? They look fine on my preview. 🤔 I will double check.

Reply:
This is what I see: 

<img width="983" alt="image" src="https://github.com/validmind/documentation/assets/15148011/b4c51640-b4b0-4644-a2a4-b8181ef2e3e8">

Reply:
Very strange. I will go ahead and add the spaces in a suggestion.

Reply:
This markup renders correctly for me:

```
| {{< var vm_framework >}} |  `{{<var vm_framework>}}` |
| {{< var vm_platform >}} |  `{{<var vm_platform>}}` |
| {{< var vm_dev >}} |  `{{<var vm_dev>}}` |
| {{< var vm_ui >}} |  `{{<var vm_ui>}}` |
```